src/main.go

```diff
@@ -216,11 +217,15 @@ func main() {
 		log.Fatalf("Failed to create backup storage directory: %v", err)
 	}
 
-	// Setup cron for monthly backups
+	// Setup cron for backups
 	c := cron.New()
-	c.AddFunc("0 0 1 * *", func() {
+	cronSchedule := config.Backup.CronSchedule
+	if _, err := cron.ParseStandard(config.Backup.CronSchedule); err != nil {
+		log.Fatalf("Invalid Cron Expression %v", err)
+	}
+	c.AddFunc(cronSchedule, func() {
 		if _, err := createBackup(); err != nil {
-			log.Printf("Monthly backup failed: %v", err)
+			log.Printf("Backup failed: %v", err)
```

Comment:
While this is great and works i think we can enhance this even more by making automated backups entirely optional